src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/alignment/AlignedContigGenerator.java

```diff
@@ -10,4 +34,213 @@
 public abstract class AlignedContigGenerator {
```

Comment:
It seems like this class should really just be an interface? The two subclasses could be top-level classes that just implement the interface. It's a bit weird to have this be an abstract class when it really only defines the signature of one method.

Reply:
Agree. But in 2nd thought, this is not related to this fix, so I'll do the refactor and the move in another PR.
I'm trying to put the implementations in one place and see if I can have one concrete class, just to avoid the two code path that exist mostly for historical reasons.